jabgui/src/main/java/org/jabref/gui/consistency/ConsistencyCheckAction.java

```diff
@@ -53,12 +52,13 @@ public BibliographyConsistencyCheck.Result call() throws Exception {
 
                 Optional<BibDatabaseContext> databaseContext = stateManager.getActiveDatabase();
                 if (databaseContext.isEmpty()) {
-                    throw new IllegalStateException((Localization.lang("No library present")));
+                    throw new JabRefException("No library present.");
```

Comment:
This Exception has to be removed completly. Not having a library opened is not an exceptional state but a legal state of jabref. If the user clicks around or for any other reason the consistency check is run in the gui, this should NOT throw an exception. Exceptions are for exceptional states only.
Please remove, make a simple dialog message maybe and write sthg to debug log.

Reply:
`dialogService.notify` should be enough --> reason: We don't want to break the users flow and try to use as less dialogs as possible.

(We are aware that these toast messages should be collected somehow, but this is future work)